### DISCUSSION TITLE
MSound (LPT sound device with DSP) emulation?

### DISCUSSION THREAD

**Discussion by Torinde** [dosbox-staging/dosbox-staging#3902] [General]
[MSound](https://www.mobygames.com/attributes/attribute/46/) - 9 games 1989-1993 - more than a simple DAC, however; a DSP contained within the box could be programmed to emulate mono, stereo, or quad stereo output, up to 48KHz.
- [Countdown](https://www.mobygames.com/game/1978/countdown/)
- [Dracula Unleashed](https://www.mobygames.com/game/460/dracula-unleashed/)
- [Links: The Challenge of Golf](https://www.mobygames.com/game/492/links-the-challenge-of-golf/)
- [Mah Jongg -V-G-A-](https://www.mobygames.com/game/70874/mah-jongg-v-g-a-/)
- [Martian Memorandum](https://www.mobygames.com/game/222/martian-memorandum/)
- [SimEarth: The Living Planet](https://www.mobygames.com/game/1835/simearth-the-living-planet/)

**Comment by johnnovak** [comment]
Extremely unlikely we'll ever add support for this (all games on the very short list have much better sound options).
